### PR TITLE
Update test.txt

### PR DESCRIPTION
**Pull request by jeppungbackup**
Add tes 6

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ Test 2
 Test 3
 Test 4 - Private
 Test 5 - Private
+Tes 6 - Prvate
```
Comment (jeppung) [inline test.txt]: Aware the typo text

**Review by jeppung — CHANGES_REQUESTED** [review]
Please fix the typo

**Comment by jeppungbackup** [comment]
Fix done

**Review by jeppung — APPROVED** [review]
Done, fix

**Pull request merged**